sw/device/tests/entropy_src_smoketest.c

```diff
@@ -47,7 +47,10 @@ bool test_main() {
       .route_to_firmware = true,
       .sample_rate = 2,
       .lfsr_seed = 2,
-  };
+      .fw_override = {
+          .enable = false,
+          .entropy_insert_enable = false,
+      }};
```

Comment:
nit: should this also set `.buffer_threshold = kDifEntropyFifoIntDefaultThreshold` as in `entropy_testutils.c ` for the sake of completeness?